sdk/python/kubeflow/training/api/training_client.py

```diff
@@ -861,27 +924,58 @@ def get_job_logs(
                                 break
 
                             # Print logs to the StdOut
-                            print(f"[Pod {pods[index]}]: {logline}")
+                            print(f"[Pod {pods[index].metadata.name}]: {logline}")
```

Comment:
Thoughts on colorizing by colorizing by pod name? Probably overkill / can be a separate PR.

Reply:
@droctothorpe Can you elaborate here ? What are your ideas on printing logs for pods?

Reply:
Here is an example of output from https://github.com/stern/stern.

![image](https://github.com/kubeflow/training-operator/assets/24783969/1e885b5a-f21a-4b58-91b8-35e0818c8036)

Each pod gets a distinct color, which helps parse a giant wall of interleaved logs. Happy to tackle it in a discrete PR.

Reply:
I see, thanks for sharing.
Can you create an issue to discuss this ?
We might need to discuss how various output will produce the results while running this (e.g. Jupyter Notebook, VSCode, local terminal).

Reply:
https://github.com/kubeflow/training-operator/issues/1981